src/Neo/Extensions/BigIntegerExtensions.cs

```diff
@@ -9,8 +9,6 @@
 // Redistribution and use in source and binary forms with or without
 // modifications are permitted.
 
-#if !NET5_0_OR_GREATER
```

Comment:
Why @shargon ? this is built for `dotnet` `7.0` too. We should of did something like that.

Reply:
Now try building for `net7.0` version

Reply:
> Why @shargon ? this is built for `dotnet` `7.0` too. We should of did something like that.

because otherwise we have different logic for dotnet 7 and netstandard.